src/components/Header/NavItems.svelte

```diff
@@ -16,7 +16,7 @@
 <!-- <a class={`underline-on-hover after:bg-${textColor}`} href="/#">Our Technology.</a> -->
 <a
   class={`underline-on-hover after:bg-${textColor}`}
-  href="https://docs.frequency.xyz/"
+  href="http://localhost:3000/"
```

Comment:
Assuming the story mean to launch local dev link than hosted version for development only

Reply:
It should still be docs.frequency.xyz there. We don't have PR deployments for docs right now.